.github/workflows/ci-cd-yt01.yml

```
@@ -48,12 +59,24 @@
+    steps:
+      - name: Set GitHub variable for latest deployed infra version
+        run: |
+          gh variable set LATEST_DEPLOYED_INFRA_VERSION --body "${{ needs.get-current-version.outputs.version }}" --env yt01 --repo ${{ github.repository }}
+
   deploy-apps:
     name: Deploy apps to yt01
     needs:
       [get-current-version, check-for-changes, deploy-infra, publish]
     # we want deployment of apps to be dependent on deployment of infrastructure, but if infrastructure is skipped, we still want to deploy the apps
-    if: ${{ always() && !failure() && !cancelled() && (github.event_name == 'workflow_dispatch' || needs.check-for-changes.outputs.hasBackendChanges == 'true') }}
+    if: ${{ always() && !failure() && !cancelled() && (github.event_name == 'workflow_dispatch' || needs.check-for-changes.outputs.hasBackendChanges == 'true' }}
```

actionlint

unexpected end of input while parsing closing ')' of nexted expression (...). expecting ")"

(expression)